server/src/main.rs

```diff
@@ -45,13 +51,38 @@ pub struct Context {
 
 /** Public Functions **/
 fn main() {
+    
+
+    // A layer that logs events to a file.
+    let file = File::create("debug.log");
+    let file = match file  {Ok(file) => file,Err(error) => panic!("Error: {:?}",error),};
+    let debug_log = tracing_subscriber::fmt::layer()
+        .with_writer(Arc::new(file));
+    
+    // A layer that logs events to stdout.
+
```

Comment:
There should not be a newline between comments and what they relate to.

Reply:
Fixed